src/components/AqiGauge/InvalidSensor.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useState} from 'react';
+import {Box, Text, Link} from '@chakra-ui/react';
+import {firestore} from '../../firebase';
+import {formatDate, formatTime} from '../Util/Dates';
+import {useTranslation} from 'react-i18next';
+
+/**
+ * Props for the InvalidSensor component that displays in the AQI Gauge box
+ * when a gray "invalid" sensor is selected
+ * - `sensorDocId` is the document ID of the selected sensor
+ */
+interface InvalidSensorProps {
+  sensorDocId: string;
+}
+
+/**
+ * This component is used in the AQI Gauge UI component. When a gray "invalid"
+ * sensor is clicked, instead of showing the AQI Gauge we display a message
+ * that the sensor is down and give the last time of valid reading
+ */
+const InvalidSensor: ({sensorDocId}: InvalidSensorProps) => JSX.Element = ({
+  sensorDocId,
+}: InvalidSensorProps) => {
+  const [lastValidDate, setLastValidDate] = useState('');
+  const [lastValidTime, setLastValidTime] = useState('');
+
+  const {t} = useTranslation('dial');
+
+  if (sensorDocId) {
+    const docRef = firestore.collection('sensors').doc(sensorDocId);
```

Comment:
This data is included in the `current-reading/sensors` doc. Use that doc instead because otherwise only admin users will be able to see it (since only admins can read the `sensors` collection).